src/components/dialogs/manage-transaction/components/account-field.vue

```diff
@@ -15,14 +16,15 @@
       </form-row>
 
       <form-row>
-        <select-field
+        <select-field-lib
           label="To account"
           placeholder="Select account"
           :values="filteredAccounts"
           label-key="name"
+          value-key="id"
           :disabled="disabled || toAccountDisabled"
           :model-value="toAccount"
-          @update:model-value="emit('update:to-account', $event)"
+          @update:model-value="updateToAccount"
```

Comment:
"Add new account" should also be present in both of these fields. Also since select-field component is not used anymore, remove `-lib` suffix entirely

Reply:
Also, in fact, we don't have many select-fields in the app, so use new component everywhere and delete unused one

Reply:
"remove -lib suffix entirely"

Maybe you mean that i need to delete basic select-field, not the one with the suffix "-lib"?